mmcv/runner/base_module.py

```diff
@@ -1,21 +1,49 @@
 # Copyright (c) Open-MMLab. All rights reserved.
+import copy
 import warnings
 from abc import ABCMeta
+from collections import defaultdict
 
 import torch.nn as nn
 
-from mmcv import ConfigDict
+from mmcv.runner.dist_utils import master_only
+from mmcv.utils.logging import get_logger, logger_initialized, print_log
+
+
+def update_init_info(module, *, init_info):
```

Comment:
Why do you adopt `*` here? If you want a keyword-only manner, `init_info` should be a keyword argument.